.github/workflows/test_bwc.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   Build-ff-linux:
     strategy:
       matrix:
-        java: [11,17,21]
```

Comment:
Shouldn't we still run bwc tests on 11 and 17?

Reply:
Not on 3.x (note this PR will not be backported):
 - we don't currently have any configuration to use different JDK versions per OpenSearch version, and this would cause combinatorial explosion in the matrix of different versions and different JDKs to test against even if we did, without very much real value anyway. 
 - currently we don't even do major version update BWCs, see discussion on #710